homeassistant/components/geo_location/__init__.py

```diff
@@ -43,6 +43,11 @@ def state(self):
             return round(self.distance, 1)
         return None
 
+    @property
+    def source(self) -> Optional[str]:
+        """Return source value of this external event."""
+        return None
```

Comment:
Let's make this mandatory by raising NotImplementedError

I don't want the automation trigger to not be able to work on all geo platforms.

Reply:
OK